packages/core/src/amazonq/explorer/amazonQTreeNode.ts

```diff
@@ -60,7 +68,17 @@ export class AmazonQNode implements TreeNode {
         return ''
     }
 
+    public isAmazonQInstalled(): boolean {
```

Comment:
This could live in this file: https://github.com/aws/aws-toolkit-vscode/blob/46560e70b7dd29d83ff557b1db246db5091dd7fe/packages/core/src/shared/utilities/vsCodeUtils.ts#L32

It will be needed by Toolkit, so putting it in a common file is helpful